CHANGELOG.md

```diff
@@ -4,6 +4,27 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## [v4.0.1](https://github.com/voxpupuli/puppet-virtualbox/tree/v4.0.1) (2020-03-18)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-virtualbox/compare/v4.0.0...v4.0.1)
+
+**Implemented enhancements:**
+
+- Checksum handling [\#40](https://github.com/voxpupuli/puppet-virtualbox/issues/40)
```

Comment:
Implemented in this release?? Maybe the label just needs removing.

Reply:
I added a tag to skip CHANGELOG

Reply:
or if was actually fixed years ago, even skip-changelog. (If you know which release it was actually fixed in, you could create a matching milestone and then assign the issue to that. Overkill though, but I think that's the way of forcing github-changelog-generator to assign an issue to the correct tag)